docs/source/download.rst

```diff
@@ -165,7 +164,6 @@ On Ubuntu, the following apt packages are required:
 - libblas-dev
 - liblapack-dev
 - libopenmpi-dev
-- libspatialindex-dev
```

Comment:
If you're installing from individual components (and not using the install script) then you do still need spatialindex somehow I think.